### DISCUSSION TITLE
Status of unity-based games with dxvk

### DISCUSSION THREAD

**Discussion by Random-Scientist** [The-Wineskin-Project/WineskinServer#231] [Q&A] [answered]
I'm trying to run Nebulous: Fleet Command (an extremely recent Unity game) with wineskin. I have dxvk installed, and Nebulous_dxgi reports that everything is normal on the dxvk side (no wire VK_KHR_surface stuff for once), but Unity still reports that I do not have a "dx11" graphics card installed. Nebulous_dx11.log's contents are as follows: (there are no other logs that contain anything relevant, unfortunately.)
<details>
<summary>Nebulous_DX11.log</summary>
<br>
`info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
warn:  D3D11CreateDevice: Unsupported driver type
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
warn:  D3D11CreateDevice: Unsupported driver type
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
warn:  D3D11CreateDevice: Unsupported driver type
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported
warn:  D3D11CreateDevice: Unsupported driver type
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_1
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
err:   D3D11CoreCreateDevice: Requested feature level not supported`
</details>

**Comment by Gcenx** [comment, marked as answer]
Unity games are notetorius for needing transform feedback and geometry shaders things that MoltenVK/Metal do not support.

Until Apple implements these feature into Metal or MoltenVK emulates then (at a great cost) there’s nothing downstream projects can do.
